replica/table.cc

```diff
@@ -4536,7 +4536,6 @@ future<> table::cleanup_tablet(database& db, db::system_keyspace& sys_ks, locato
     co_await stop_compaction_groups(sg);
     co_await utils::get_local_injector().inject("delay_tablet_compaction_groups_cleanup", std::chrono::seconds(5));
```

Comment:
> * We already hold erms during replica-side request execution, but in
an ad-hoc, non-systemic way in lower layers of storage_proxy
(e.g. in sp::mutate_locally and do_query_tablets).

We need to propagate erms down from the point of acquisition, to make sure we always operate with the same topology.

We can merge those fixes later on, but we need to do them.